keps/prod-readiness/sig-scheduling/5194.yaml

```diff
@@ -0,0 +1,3 @@
+kep-number: 5194
+alpha:
+  approver: "@johnbelamaric"
```

Comment:
Probably best to pick someone else since I will be out all but two days between now and KEP freeze

Reply:
Yes - I can take it